src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -6,12 +6,23 @@
 #include <string>
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
 #include "DataStructures/Tensor/TypeAliases.hpp"
 #include "Evolution/Systems/GeneralizedHarmonic/TagsDeclarations.hpp"
 
 class DataVector;
 
 namespace GeneralizedHarmonic {
+namespace Tags {
+/*!
+ * \brief Evolved variable which is the spacetime metric, i.e. \f$\psi_{ab}\f$
+ */
+template <size_t Dim, typename Frame>
+struct Psi : db::SimpleTag {
```

Comment:
Isn't this tag redundant with gr::Tags::SpacetimeMetric? Why not just use gr::Tags::SpacetimeMetric? Is there a problem with CharSpeed<gr::Tags::SpacetimeMetric> or something?

Reply:
It is. But it seems logically consistent to keep separate tags for characteristic fields, because this may be so for most other evolution systems in general, and then a few of them will have `N(Char fields tags) == N(Fund field tags)` and a few with `N(Char fields tags) < N(Fund field tags)` which is something another user may assume not to be way down the line. does that make sense, or are evolution systems not to be combined?

Reply:
This makes sense, as long as there's some efficient way to make sure in cases where they are redundant, the tags point to the same thing.

Reply:
No, please don't do this.

Reply:
If there isn't a great way to make them be the same (like one is the alias to another), then I agree with @nilsdeppe, and I still think that you should use `CharSpeed<gr::Tags::SpacetimeMetric>` instead of creating a tag `Psi`

Reply:
> "No" as in "No, there really isn't a great way to make them be the same. We should have one simple tag for one thing. This is why hydro shares primitive tags in `PointwiseFunctions/Hydro/Tags.hpp`"

@nilsdeppe - For my understanding, why is `template<size_t Dim, typename Frame>
using UPsi = gr::Tags::SpacetimeMetric<Dim, Frame, DataVector>;`  not a good idea?

Reply:
@geoffrey4444  - `Psi` has been removed. Now this code uses `gr::Tags::SpacetimeMetric` for it everywhere.